internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -74,6 +75,8 @@ const (
 	MACHINE_TYPE                        = "machine.openshift.io/cluster-api-machine-type"
 	MCS_CERT_NAME                       = "ca.crt"
 	OPENSHIFT_MACHINE_API_NAMESPACE     = "openshift-machine-api"
+	CustomSecretLabelName               = "some-custom-label-for-assisted-service-secrets"
```

Comment:
```suggestion
	WatchResourceLabel               = "agent-install.openshift.io/watch"
```

We could move this to `common.go` and use it for `ClusterDeploymentController`